backend/internal/controllers/book.go

```diff
@@ -47,6 +56,24 @@ func (c *BookController) GetBook(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// userID empty if and only if signed-in user makes get book request
+	if userID != "" {
+
+		favorited_id, err := c.Books.FetchFavoritedBook(req.Context(), userID, bookID)
```

Comment:
this should probably just return a (bool, error) -> true if bookID is favorited by userID and false if not